### ISSUE TITLE
Hyper stopped working tottaly

### ISSUE THREAD

**Issue by dankonino** [zeit/hyper#1488]
Hello, I'd like to present this "bug"?  I had latest version of hyper term which suddenly is going just displaying the name next to the apple logo in the bar, no file, edit nothing. I tried to redownload the newest versions. I am assuming it happened because some miscommunication while using https://www.npmjs.com/package/hpm-cli for plugins. Is there please a way to reinstall or reconfigure Hyper to default values? I just don't know where is it located

**Comment by jamo** [comment]
IIRC the only config file for hyper is: `~/.hyper.js` and the installed plugins are located in `~/.hyper_plugins`

**Comment by dankonino** [comment]
Yeaaaaaahhhhhh thanks :) it's running after two weeks my <3     "hyperterm-transparent-bg"
this plugin was causing that stuuuuf

**Comment by dankonino** [comment]
Yeaaaaaahhhhhh thanks :) it's running after two weeks my <3

**Comment by jamo** [comment]
btw, at least on macOs the transparency has been natively supported (without any plugins) for a while now. Just make sure the bg color is transparent e.g `backgroundColor: 'rgba(37, 37, 37, 0.8)',`

**Comment by dankonino** [comment]
lovely !  thanks for the info.  it just seems not working for me now, I'll try to play with that.

**Comment by pujanTandukar** [comment]
I just downloaded Hyper and I was messing around with some configuration, but it seemed to be working fine. After adding "hyperpower" and restarting the app, it crashed and I'm having the same issue, It displays the name next to the apple logo but all it has is "Quit". I cannot seem to open any new windows or anything at all. I tried reinstalling the app, changing the .hyper.js file to the original but it just doest seem to work. I have tried restarting my computer as well.
